regolith/schemas.py

```diff
@@ -2818,42 +2843,54 @@
         },
         "actions": {
             "description": "action items expected from the group members for that particular meeting week",
-            "required": False,
+            "required": True,
             "type": "list",
         },
         "agenda": {
             "description": "schedule of the current meeting",
-            "required": False,
+            "required": True,
             "type": "list",
         },
         "buddies": {
             "description": "list of pairs of group members that are selected for the buddy round robin",
-            "required": False,
+            "required": True,
             "type": "list",
         },
         "day": {
```

Comment:
as well as day month and year, I think we need date.  We need both because we are switching over from one to the other but we want to maintain backwards compatibility.  Please check date is there.  Thanks.

Reply:
Done: added a date property